PatientRegistrationApp/src/App.vue

```diff
@@ -159,36 +159,52 @@
 					<v-footer absolute color="white">
 						<template v-if="isGreetingPage()">
 							<v-spacer></v-spacer>
-							<v-icon large color="secondary" @click="goToNextPage()">
-								mdi-chevron-right
-							</v-icon>
+							<v-btn color="secondary" class="ma-2 white--text" @click="goToNextPage()">
```

Comment:
More appearance here, but when at 100% zoom the ethnicity input field appears cut off. You can still input stuff normally but it looks funky... This looks fine when at 75% zoom however. 

![image](https://user-images.githubusercontent.com/67079030/100824679-47026500-341c-11eb-9873-a92cc86acffb.png)

Reply:
Good catch!  That is fixed and I checked all of the pages